### DISCUSSION TITLE
Convert R10 DXF to R12 DXF

### DISCUSSION THREAD

**Discussion by ttshaw1** [mozman/ezdxf#452] [Q&A]
Quick question, is there a way to convert an existing DXF from one version to another without using the ODA File Converter addon (proprietary)? I wasn't able to find anything in the documentation about this aside from the ODA info, but I (perhaps naively) think this is simple enough that there probably is a way to do it.

Thanks!

**Comment by mozman** [comment]
Loading and saving the R10 DXF with ezdxf should create a R12 DXF file. If this doesn't work with your files, you can upload this files in a zip archive and I will try to add the missing features.

**Reply by ttshaw1** [reply]
```
file = ezdxf.readfile('r10_input.dxf')
file.saveas('ezdxf_output.dxf')
```
gives an output file with ACADVER 1009 as you say. I'll make sure this file works on our temperamental tool in the next couple of days and then mark this as the answer. Thank you very much.
